mmv1/third_party/terraform/tests/resource_os_config_os_policy_assignment_test.go.erb

```diff
@@ -0,0 +1,147 @@
+<% autogen_exception -%>
+
+package google
+
+import (
+	"context"
+	"fmt"
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	osconfig "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/osconfig<%= dcl_version(version) -%>"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"strings"
+	"testing"
+)
+
+func TestAccOsConfigOsPolicyAssignment_PercentOsPolicyAssignment(t *testing.T) {
+	// Tests fail due to long rollout times
+	skipIfVcr(t)
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_name":  getTestProjectFromEnv(),
+		"zone":          getTestZoneFromEnv(),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckOsConfigOsPolicyAssignmentDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccOsConfigOsPolicyAssignment_PercentOsPolicyAssignment(context),
+			},
+			{
+				ResourceName:            "google_os_config_os_policy_assignment.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"rollout.0.min_wait_duration"},
+			},
+		},
+	})
+}
+
+func testAccOsConfigOsPolicyAssignment_PercentOsPolicyAssignment(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_os_config_os_policy_assignment" "primary" {
+  instance_filter {
+    all = false
+    exclusion_labels {
+      labels = {
+        label-two = "value-two"
+      }
+    }
+    inclusion_labels {
+      labels = {
+        label-one = "value-one"
+      }
+    }
+    inventories {
+      os_short_name = "centos"
+      os_version    = "8.*"
+    }
+  }
+
+  location = "europe-west1-a"
+  name     = "tf-test-assignment%{random_suffix}"
+
+  os_policies {
+    id   = "policy"
```

Comment:
possibly add allow_no_resource_group_match and description in here to get full coverage?